src/App.js

```diff
@@ -1,6 +1,27 @@
-import Calculator from './components/Calculator';
 import './components/calculator.css';
+import './components/quotes.css';
+import './lauout/layout.css';
+import './pages/calculatorpage.css';
+import './pages/quotespage.css';
+import './pages/home.css';
+import {
+  createBrowserRouter, Route, createRoutesFromElements, RouterProvider,
+} from 'react-router-dom';
+import Calpage from './pages/CalculatorPage';
+import Quotepage from './pages/Quotespage';
+import Home from './pages/Home';
+import Layout from './lauout/layout';
+
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route element={<Layout />}>
+      <Route path="/" element={<Home />} />
```

Comment:
- [optional] Great job following the wireframe and design. I will suggest you add more styling (colors) to the home and quotes page to advance the look and feel. Also, you can highlight the navigation links, so they indicate the current page which the user is in. nice work.